tests/app/test_jinja_filters.py

```diff
@@ -202,12 +202,17 @@ def test_radio_class_visible_attribute():
                     "label": "Other",
                     "value": "Other",
                     "detail_answer": {
-                        "mandatory": False,
-                        "id": "other-answer",
-                        "label": "Please enter a number of items",
-                        "type": "Number",
-                        "parent_id": "radio-question-numeric-detail",
+                        "type": "Dropdown",
+                        "id": "mandatory-checkbox-with-mandatory-dropdown-detail-answer",
+                        "mandatory": True,
                         "visible": True,
+                        "label": "Please specify heat level",
+                        "placeholder": "Select heat level",
+                        "options": [
+                            {"label": "Mild", "value": "Mild"},
+                            {"label": "Medium", "value": "Medium"},
+                            {"label": "Hot", "value": "Hot"},
+                        ],
```

Comment:
I can't see why this test was changed to a dropdown when it's the visible property that is being tested?

Reply:
It was simply to satisfy 1 line of coverage to avoid writing a suite of integration tests when there are functional tests for it. Didn't think to add more jinja filter tests since none currently exist to test the config anything but like an individual attribute. I could technically add new tests to test the full context.

Reply:
Just cover the ```if answer_type == "Dropdown":` block.

Reply:
Can't we just have an additional specific test for other dropdown answers?

Reply:
Ya my point was we dont have specific tests for testfield/numeric etc, only testing the "open" attribute really. Let me refactor the existing ones to reuse the same partial json and write some to test each variant.

Reply:
Added some tests to test the full `OtherConfig` independently, let me know what you think.